docs/user-guides/retriever.md

```diff
@@ -1,4 +1,4 @@
-# Search API
```

Comment:
Can you rename this doc to `search.md`?
And maybe change the others: `Client API->CLIP Client` and `Server API->CLIP Server`

Reply:
nope, I will reject this comment. Let's focus on search-related content in this PR.